modules/subprocess/src/main/scala/notebook/kernel/pfork/BetterFork.scala

```diff
@@ -170,7 +169,7 @@ object BetterFork {
         } else {
           acc ++ (cl.asInstanceOf[URLClassLoader].getURLs map { u =>
             val f = new File(u.getFile)
-            URLDecoder.decode(f.getAbsolutePath, "UTF8")
+            URLDecoder.decode(f.getParent, "UTF8")
```

Comment:
So, here we'll change all `<path>/library-one.jar` into `<path>`, right?

Because, below you map all libs to `<path>/*`. Wouldn't that be a bit too aggressive? Just wondering, of course.
Maybe could you elaborate very shortly why you did that (I guess it is for the path length)? So we can see if it'll be reliable or if it could be done elsewhere (like in the native packager script generator) or of course simply kept as is.